src/main/cpp/t07_ip.cpp

```diff
@@ -34,5 +34,46 @@
 using namespace std;
 
 int t07_ip() {
+	int p1, p2, p3, p4;
+    int cou = 0;
+    int pos = 0;
+    int arr[4];
+    for(int i = 0; i < 4; i++)
+        arr[i] = -1;
 
+    p1 = p2 = p3 = p4 = -1;
+    string s;
+    getline(cin, s);
+
+    while((int)s.find(".", pos) > -1){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Inefficient usage of string::find() in condition; string::starts_with() could be faster.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513044)